src/secureCreatives.js

```diff
@@ -53,9 +53,12 @@ function receiveMessage(ev) {
 }
 
 function sendAdToCreative(adObject, remoteDomain, source) {
-  const { adId, ad, adUrl, width, height } = adObject;
-
-  if (adId) {
+  const { adId, ad, adUrl, width, height, renderer } = adObject;
+  // rendering for outstream safeframe
+  if (renderer && renderer.url) {
```

Comment:
should we move this logic into a separate function so it sits in one place? For example, it's also in pbjs.renderAd().